test/address.test.ts

```diff
@@ -0,0 +1,196 @@
+import axios from 'axios';
+import { encryptPassword } from '../src/data/graphql/password';
+import { connectDb, connectServer } from './util/connect.util';
+import { disconnectServer, disconnectDb } from './util/disconnect.util';
+import { expect } from 'chai';
+import { createUser, findUserByEmail, deleteAllUsers } from '../src/data/user/user.db.datasource';
+import { deleteAllAddresses, getUserAddresses, createAddress } from '../src/data/address/address.db.datasource';
+
+const port = process.env.PORT;
+const url = `http://localhost:${port}/`;
+
+describe.only('address testes', () => {
+  const mutation = `
+        mutation createAddress($userId: Int!, $data: AddressInput!){
+            createAddress(userId: $userId, data: $data) {
+                id
+                cep
+                street
+                streetNumber
+                complement
+                neighborhood
+                city
+                state
+            }
+        }
+    `;
+
+  let userId: number;
+
+  before('before', async () => {
+    const user = {
+      data: {
+        name: 'Sam',
+        email: 'sam@example.com',
+        password: await encryptPassword('Sam123'),
+        birthDate: '09-04-2004',
+      },
+    };
+    await connectServer();
+    await connectDb();
+    await createUser(user);
+    const sam = await findUserByEmail('sam@example.com');
+    if (sam) {
+      userId = sam.id;
+    }
+  });
+
+  after('after', async () => {
+    await disconnectServer();
+    await deleteAllAddresses();
+    await deleteAllUsers();
```

Comment:
Essas funções que limpam as tabelas deveriam estar num afterEach não?
Dessa forma que está vai ser limpo apenas uma vez após o describe todo ser rodado, e como um teste não pode depender do dado de outro teste ele deveria zerar o banco a cada it.